docs/StandardOptionalAttributes.rst

```diff
@@ -170,6 +170,103 @@ By default, OpenEXR files have the following attributes:
 
   NDC space in OpenEXR is the same as in Pixar's Renderman.
 
+**sensorCenterOffset**
+  Horizontal and vertical distances, in microns, of
+  the center of the light-sensitive area of the camera's sensor from
+  a point on that sensor where a sensor surface normal would intersect
+  the center of the lens mount.
+
+  When compared to an image captured with a perfectly centered sensor,
+  an image where both horizontal and vertical distances were positive
+  would contain more content holding what was at the right and what was
+  at the bottom of the scene being captured.
+
+**sensorOverallDimensions**
+  Dimensions of the light-sensitive area of the sensor, in millimeters,
+  independent of the subset of that region from which image data are
+  obtained.
+
+**sensorPhotositePitch**
+  Ddistance between centers of sensor photosites, in microns.
+
+**sensorAcquisitionRectangle**
+  The rectangular area of the sensor containing photosites
+  the contents of which are in one-to-one correspondence with
+  the captured sensels, for a monochrome sensor, or with the
+  reconstructed RGB pixels, for a sensor covered with color
+  filter array material in a Bayer or a similar pattern.
+
+  Because understanding the above formal definition is
+  critical for many applications, including camera solvers,
+  some short definitions:
+  - a *photosite* is that optoelectronic component on the
```

Comment:
Insert a blank line before each of the list entries beginning with "-". The sphinx formatting of the .rst is erroring, confused by the indentation.